staging-service/src/main/java/com/github/cafdataprocessing/services/staging/dao/filesystem/SubBatchWriter.java

```diff
@@ -20,8 +20,8 @@
 import com.github.cafdataprocessing.services.staging.exceptions.IncompleteBatchException;
 import com.github.cafdataprocessing.services.staging.exceptions.InvalidBatchException;
 import com.github.cafdataprocessing.services.staging.exceptions.StagingException;
+import com.github.cafdataprocessing.workers.document.schema.validator.InvalidDocumentException;
```

Comment:
I think it was in the correct location where it was.

Reply:
We have removed 'microfocus' from package names in worker-document and moved it here: https://github.com/CAFDataProcessing/worker-document/blob/develop/worker-document-validator/src/main/java/com/github/cafdataprocessing/workers/document/schema/validator/InvalidDocumentException.java

Reply:
I meant alhabetically speaking.